packages/authhero/src/types/EmailService.ts

```
@@ -0,0 +1,12 @@
+export type SendEmailParams = {
+  to: string;
+  from: string;
+  subject: string;
+  html?: string;
+  text?: string;
+};
+export type SendEmailResponse = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)